packages/authenticator/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -419,6 +425,12 @@ class Authenticator extends StatefulWidget {
   /// method.
   final AuthenticatorStep initialStep;
 
+  /// The default dial code code to use for phone number sign in or sign up.
+  /// The default is set to US (+1).
+
+  /// The options to configure the dial code selector.
```

Comment:
Use a `///` to separate lines:

```suggestion
  /// The default dial code code to use for phone number sign in or sign up.
  /// The default is set to US (+1).
  ///
  /// The options to configure the dial code selector.
```

Reply:
I would rephrase this a bit, since it's about the options as a whole now.

```suggestion
  /// The options for displaying dial codes in popup modals.
  ///
  /// Phone number fields in the Authenticator allow the user to select a dial code
  /// via a popup modal different than the default. By default, the US dial code (+1)
  /// is chosen and a list of all available dial codes is shown in the popup.
  ///
  /// Use these options to select which dial code is selected by default and, optionally,
  /// limit which options are shown in the modal.
```